src/index.css

```diff
@@ -15,12 +15,43 @@
   height: 100vh;
 }
 
+nav {
+  font-family: "Poppins", sans-serif;
```

Comment:
- [ ] [OPTIONAL] Good job on following the design guidelines wireframe for styling your app. I encourage you to add more styles in order to improve the look and feel and to make it your own.